crates/sui-adapter/src/adapter.rs

```diff
@@ -76,6 +76,9 @@ pub fn new_move_vm(natives: NativeFunctionTable) -> Result<MoveVM, SuiError> {
                 max_generic_instantiation_length: Some(MAX_GENERIC_INSTANTIATION_LENGTH),
                 max_function_parameters: Some(MAX_FUNCTION_PARAMETERS),
                 max_basic_blocks: Some(MAX_BASIC_BLOCKS),
+                max_value_stack_size: MAX_VALUE_STACK_SIZE,
+                max_type_nodes: Some(256),
```

Comment:
Nit: could you move this to protocol_constants?

Reply:
I already created a constant but did not use it... Will fix!